pods/covid-spain-chart/base.css

```diff
@@ -0,0 +1,293 @@
+/*
```

Comment:
right now we don't need this style sheet (is just to mimic Github Styles), buuuuut THINKING LOUD... we could just add the descriptions using markdown and use this as base style for the markdown viewer